### PR TITLE
docs: Add process - "How to create a PR"

### PR DESCRIPTION
**Pull request by gullapalli9482**
## 🛠️ Fixes Issue

closes #56

### CODE REVIEW

**Comment by gullapalli9482** [comment]
I think I did same mistake again,just check it out once , else I'll have to start from beginning again

**Comment by KendallDoesCoding** [comment]
> I think I did same mistake again,just check it out once , else I'll have to start from beginning again

No, doesn't look like you did the mistake, 2 files have been changed and there are no merge conflicts.

**Review by Pradumnasaraf — CHANGES_REQUESTED** [review]
Thank you for raising the PR, You don't need to close this PR, you can simply fix this by deleting the file which is not required, I think it's `pages/How-to/guide/2`, and commit it to the same branch from which you created PR. The PR will auto fetch those changes.

If you have doubts feel free to reach out :)

__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix

**Comment by gullapalli9482** [comment]
I hope, I deleted the correct file!

**Comment by gullapalli9482** [comment]
Any comments @Pradumnasaraf

**Comment by Pradumnasaraf** [comment]
So sorry for the delay, I was busy with exams and other stuff 😞

File: pages/How-to/guide/creating-pr.md [hunk 1]
```diff
@@ -1,4 +1,73 @@
...
-</div>
+A pull request is a method for submitting contributions to a repository, with some extra changes made by the individual. 
+Now there are two ways of creating a pull request, the procedure is mostly the same the only thing that differs is the method. A pull request could either be made with the help of using the github UI or making changes locally on your machine via terminal.
+We are going to cover both of these methods here.
+
+Let's start with general procedure of creating a pull request:-
+1) Creating a fork of the repository is essential. Now what does a fork mean, it is nothing but a clone of the repository where you are free to make your to make respective changes. 
+2) Where to find the fork button
+
+
```
Comment (Pradumnasaraf) [inline pages/How-to/guide/creating-pr.md:9]:
```suggestion
```

File: pages/How-to/guide/creating-pr.md [hunk 2]
```diff
@@ -1,4 +1,73 @@
...
+Now let's cover how to create a pull request via the github UI
+
+1) Now since you have already forked the project ,just go to the file you want to edit, you might a notice a pencil icon at the top right corner of the file
+
+![image](https://user-images.githubusercontent.com/75060398/172180663-0fdfe56d-02d9-422c-9528-6f2c74302eb5.png)
+
+3) Click on it, now one would be able to edit the file. After making the desired changes at the bottom of the screen you might observe an option to commit your changes
+
+
```
Comment (Pradumnasaraf) [inline pages/How-to/guide/creating-pr.md:28]:
```suggestion
```

File: pages/How-to/guide/creating-pr.md [hunk 3]
```diff
@@ -1,4 +1,73 @@
...
+![image](https://user-images.githubusercontent.com/75060398/172181128-305dfde6-2c58-419f-b0a1-6ccc60fd9863.png)
+
+
```
Comment (Pradumnasaraf) [inline pages/How-to/guide/creating-pr.md:31]:
```suggestion
```

File: pages/How-to/guide/creating-pr.md [hunk 4]
```diff
@@ -1,4 +1,73 @@
...
+After typing the command you might notice the terminal to hang for a few seconds, it would ask you details of you github account.
+After filling them in it would give you appropriate message and displays two links, the first link is the one for creating you pull request.
+Open it up in your new browser, give your pull request a proper name and some description about it,tag some issues in case it resolves any of them.
+And you have successfully created your pull request 🥳 🥳!!!!
+
+
+
+
+
```
Comment (Pradumnasaraf) [inline pages/How-to/guide/creating-pr.md:72]:
```suggestion
```

**Review by Pradumnasaraf — CHANGES_REQUESTED** [review]
Please remove all the extra blank lines
(I am suggesting some of them)

__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix

**Comment by Pradumnasaraf** [comment]
Thank you for the effort and time. Merged your PR. Good luck for your exams!!

__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :ok_hand: This code looks good

**Pull request merged**